packages/nvidia_nat_core/tests/nat/server/test_unified_api_server.py

```
@@ -346,6 +365,23 @@ async def test_generate_stream_endpoint(client: httpx.AsyncClient, config: Confi
     assert response.status_code == 200
 
 
+async def test_generate_stream_endpoint_yields_error_when_workflow_raises(client: httpx.AsyncClient, config: Config):
+    """When the streaming workflow raises, generate stream contains an Error chunk with code workflow_error."""
+
+    async def raising_gen(*args, **kwargs):
```

Ruff (0.15.0)

[warning] Unused function argument: `args`

(ARG001)